packages/shared/src/svg/SquadInviteMembers.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { ReactElement } from 'react';
+
+export default function SquadsInviteMembersSvg(
+  props: React.SVGProps<SVGSVGElement>,
+): ReactElement {
+  return (
+    <svg
+      width="420px"
+      height="215px"
+      viewBox="0 0 420 215"
+      version="1.1"
+      xmlns="http://www.w3.org/2000/svg"
+      xmlnsXlink="http://www.w3.org/1999/xlink"
+      {...props}
+    >
+      <title>Group</title>
```

Comment:
I think the markup of these SVGs can be minified, Tsahi used to assist with that.
Perhaps we can check with design how to optimize the SVGs

Reply:
Makes sense, let me take a look.

Reply:
Done 👍 

Used this app: https://jakearchibald.github.io/svgomg/